src/pages/index.js

```diff
@@ -1,256 +1,231 @@
+  profileEditButton,
+  profileTitleInput,
+  profileDescriptionInput,
+  profileAddButton,
+  newCardEditForm,
+  avatarForm,
+  changeAvatarImageButton,
+  removeCardForm,
+} from "../utils/constants.js";
 import Card from "../components/Card.js";
+import Api from "../components/Api.js";
 import FormValidator from "../components/FormValidator.js";
-import PopupWithForm from "../components/PopupWithForm.js";
-import PopupWithImage from "../components/PopupWithImage.js";
-import PopupWithConfirm from "../components/PopupWithConfirm.js";
 import Section from "../components/Section.js";
+import PopupWithForm from "../components/PopupWithForm.js";
+import "../pages/index.css";
 import UserInfo from "../components/UserInfo.js";
-import Api from "../components/Api.js";
-
-/////////////////////////////////////////////////// ELEMENTS ///////////////////////////////////////////////////////////////////////////
-
-//  PROFILE
-const avatarPicture = document.querySelector(".profile__avatar_edit-button");
-const profileEditButton = document.querySelector("#profile-edit-button");
-const profileEditModal = document.querySelector("#profile-edit-modal");
-const profileNameInput = document.querySelector("#profile-name-input");
-const profileDescriptionInput = document.querySelector(
-  "#profile-description-input"
-);
-const profileEditForm = profileEditModal.querySelector("#modal-form-1");
-const avatarEditModal = document.querySelector("#avatar-change-modal");
-const avatarEditForm = avatarEditModal.querySelector("#modal-form-3");
-
-// CARD TEMPLATE
-const cardTemplate =
-  document.querySelector("#card-template").content.firstElementChild;
-
-// CARD ADD
-const cardAddButton = document.querySelector("#card-add-button");
-const cardAddModal = document.querySelector("#card-add-modal");
-const cardAddForm = cardAddModal.querySelector("#modal-form-2");
-
-// UNIVERSAL SUBMIT BUTTON
-const submitButton = document.querySelector(".modal__button");
+import PopupWithImage from "../components/PopupWithImage.js";
 
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////// FORM VALIDATOR //////////////////////////////////////////////////////////////////////////////////////////
 
-// SECTION CLASS FOR CARDS
-const cardSection = new Section(
-  {
-    renderer: (item) => {
-      cardSection.addItem(makeCard(item));
-    },
-  },
-  ".cards"
-);
+// EDIT PROFILE
+const editProfileValidator = new FormValidator(config, editProfileFormElement);
+editProfileValidator.enableValidation();
+// ADD CARD
+const addCardValidator = new FormValidator(config, newCardEditForm);
+addCardValidator.enableValidation();
+// AVATAR
+const avatarFormValidator = new FormValidator(config, avatarForm);
+avatarFormValidator.enableValidation();
+// DELETE CARD
+const removeCardValidator = new FormValidator(config, removeCardForm);
```

Comment:
You don't need `removeCardValidator` because the confirmation modal doesn't have any **inputs**